portafly/doc/ux_specs/Audience > Accounts > Listing.md

```diff
@@ -0,0 +1,21 @@
+# Audience > Accounts > Listing
```

Comment:
I think it would be better to have a different naming, using snake case `_` instead of ` > `
Also, maybe having a directory structure like the navigation? Say... `doc/ux_specs/audience/accounts/list.md` or `doc/ux_specs/audience/accounts/index.md`

Reply:
I agree, the name of the file should avoid blank spaces, better use `_` to separate
And the directory structure may reflect navigation

Reply:
yes, that's a good point.
check https://github.com/3scale/porta/pull/1828/commits/a93a7c10497e4122a4e6ca7e1b469eae1cf52c8a